auth/store.go

```diff
@@ -156,7 +162,8 @@ type authStore struct {
 	rangePermCache map[string]*unifiedRangePermissions // username -> unifiedRangePermissions
 
 	simpleTokensMu sync.RWMutex
-	simpleTokens   map[string]string // token -> username
+	simpleTokens   map[string]string          // token -> username
+	simpleTokenInf map[string]simpleTokenInfo // token -> (username, token timer, done channel)
```

Comment:
simpleTokenDeadline map[string]time.Time and use a single go routine to scan this.